pom.xml

```diff
@@ -67,31 +67,31 @@
 
     <properties>
         <app.version>${project.version}</app.version>
-        <commons-io.version>2.8.0</commons-io.version>
+        <commons-io.version>2.13.0</commons-io.version>
         <git-commit-id-maven-plugin.version>5.0.0</git-commit-id-maven-plugin.version>
-        <hibernate5-ddl-maven-plugin.version>2.3.0</hibernate5-ddl-maven-plugin.version>
+        <hibernate5-ddl-maven-plugin.version>2.5.0</hibernate5-ddl-maven-plugin.version>
         <java.version>17</java.version>
         <jdk.source>17</jdk.source>
         <jdk.target>17</jdk.target>
-        <jquery.version>3.5.1</jquery.version>
-        <lms-canvas-rivet.version>5.2.0.0</lms-canvas-rivet.version>
-        <lms-embedded-services.version>5.2.7</lms-embedded-services.version>
-        <lms-team-spring-boot-it12>4.7</lms-team-spring-boot-it12>
+        <jquery.version>3.7.0</jquery.version>
+        <lms-canvas-rivet.version>5.2.5.1</lms-canvas-rivet.version>
+        <lms-embedded-services.version>5.2.11</lms-embedded-services.version>
```

Comment:
Waiting for latest embedded services release so that this can go to that version